lib/pavilion/cmd_utils.py

```diff
@@ -136,7 +136,7 @@ def arg_filtered_series(pav_cfg: config.PavConfig, args: argparse.Namespace,
             output.fprint(verbose, "Using default search filters: The current system, user, and "
                                    "newer_than 1 day ago.", color=output.CYAN)
             args.user = utils.get_login()
```

Comment:
This and the next change fix a bug in listing series - the series filter creator took the wrong argument types.